OtripleS.Web.Api.Tests.Acceptance/Brokers/OtripleSApiBroker.Students.cs

```diff
@@ -18,5 +18,8 @@ public partial class OtripleSApiBroker
 
 		public async ValueTask<Student> DeleteStudentByIdAsync(Guid studentId) =>
 			await this.apiFactoryClient.DeleteContentAsync<Student>($"{StudentsRelativeUrl}/{studentId}");
+
+		public async ValueTask<Student> PutStudentAsync(Guid studentId, Student student) => 
+			await this.apiFactoryClient.PutContentAsync($"{StudentsRelativeUrl}/{studentId}", student);
```

Comment:
Put doesn't require an `id` in the url, the `id` is already there in the model, you just need to pass the relative url and the student.
The update student service implementation and controller are already in the master branch, so please make sure your acceptance test is passing.